components/brave_wallet/browser/fil_tx_manager_unittest.cc

```diff
@@ -88,14 +88,18 @@ class FilTxManagerUnitTest : public testing::Test {
 
   PrefService* prefs() { return &prefs_; }
 
+  url::Origin GetOrigin() const {
+    return url::Origin::Create(GURL("https://brave.com"));
+  }
+
   void AddUnapprovedTransaction(mojom::FilTxDataPtr tx_data,
                                 const std::string& from,
                                 std::string* meta_id) {
     auto tx_data_union = mojom::TxDataUnion::NewFilTxData(std::move(tx_data));
 
     base::RunLoop run_loop;
     fil_tx_manager()->AddUnapprovedTransaction(
-        std::move(tx_data_union), from,
+        std::move(tx_data_union), from, GetOrigin(),
```

Comment:
Need to add `WalletOrigin` test too, see eth/solana_tx_manager_unittest.cc

Reply:
done

Reply:
done